manager/integration/tests/test_statefulset.py

```diff
@@ -69,7 +69,8 @@ def create_and_test_backups(api, cli, pod_info):
         for i in range(DEFAULT_BACKUP_TIMEOUT):
             backups = bv.backupList().data
             for b in backups:
-                if b['snapshotName'] == pod['backup_snapshot']['name']:
+                if b['snapshotName'] == pod['backup_snapshot']['name']\
```

Comment:
Why not just use status to check? using `url` is not quite understandable. 

```
'state': 'Completed'
```

ref: http://github.com/longhorn/longhorn-manager/blob/5b64adfaa300d2765cfeba232b883cf35b28e1bc/controller/backup_controller.go#L352-L364

Reply:
Hi @innobead,
I changed  code to use 'state' instead of 'url' which defined in CR to check backup status
And if I did not separate line, flake8 won't let pass, currently I still need to figure out how to conquer that. thank you.

Reply:
that's ok 👍